src/Void/Init.fs

```diff
@@ -5,6 +5,19 @@ open Void.Lang.Interpreter
 open Void.Lang.Editor
 open Void.ViewModel
 
+type VoidOptions = {
+    enableVerboseMessageLogging : bool

```

Comment:
I'm trying to follow the .NET convention of using capitals in the names of record fields. Would you mind capitalizing this?

I should probably throw together a minimal style guide for the repo. Generally, I've been doing:

````
TypesLikeThis
methodsLikeThis
functionsLikeThis
RecordLabelsLikeThis
TypeConstructorsLIkeThis
variablesLikeThis
ModulesLikeThis
Namespace.Like.This
```